components/mediation-ui/org.wso2.carbon.mediation.security.ui/src/main/resources/web/mediation_secure_vault/js/properties.js

```diff
@@ -315,48 +290,29 @@ function showRetention() {
 	}
 }
 
-
-
-
 function validateEmptySV(fld, fldName) {
 	var error = "";
 	fld.value = fld.value.replace(/^\s\s*/, '').replace(/\s\s*$/, '');
-	
 	if (fld.value.length == 0) {
-		error = "The required field" + " " + fldName + " " + "Not fild"
-				+ "<br />";
-	} else {
-		// fld.style.background = 'White';
+		error = "The required field " + fldName + " is not filled";
 	}
 	return error;
 }
 
 function validateForInputSV(fld, fldName) {
   	var error = "";
-	var illegalChars = /(\<[a-zA-Z0-9\s\/]*>)/; // match any starting tag
+	var illegalChars = /[~!@#$%^&*()\\\/+=\:;<>'"?[\]{}|\s,]/;
```

Comment:
Shall we add comment explaining about this regex, so that whoever goes through this code get the idea by looking at the comment itself?